apps/AppWithWearable/lib/backend/api_requests/api_calls.dart

```diff
@@ -120,8 +120,15 @@ Future<Structured> generateTitleAndSummaryForMemory(String transcript, List<Memo
   // TODO: this has to be toggled and played around
   // TODO: probably best to determine usefulness with function call before executing this prompt
   final languageCode = SharedPreferencesUtil().recordingsLanguage;
+  String storedPluginsString = SharedPreferencesUtil().storedPluginsString;
+  List<dynamic> plugins = jsonDecode(storedPluginsString);
+  List<dynamic> enabledPlugins = plugins.where((plugin) => plugin['isEnabled'] == true).toList();
+  String combinedPluginPrompt = "";
+  for (var plugin in enabledPlugins) {
+    combinedPluginPrompt += plugin['prompt'] + " "; 
+  }
   var prompt =
-      '''Based on the following recording transcript of a conversation, provide structure and clarity to the memory.
+      '''$combinedPluginPrompt Based on the following recording transcript of a conversation, provide structure and clarity to the memory.
```

Comment:
Not sure about this prompt that big, like with 2 plugins enabled there might be confusions, but let's see.